README.md

```diff
@@ -124,6 +124,54 @@ compiler. Compilation will happen automatically for the packages that need it, b
 compiler must be available. It can be installed using `apt install protobuf-compiler`
 or `brew install protobuf`
 
+### Formatters
+
+- Yaml: `yamlfmt`
+- Kotlin: `ktfmt`
+- Rust: `rustfmt`
+- Markdown: `markdownlint-cli2`
+- Protobuf: `protolint`
+
+### Install everything at once please
```

Comment:
I'm confused what this section is. You have instructions on things like installing protobuf below, when you already have it above in the protobuf section. You also mention using brew, but my understanding is that that is usually for Mac, not linux. Above you have use `apt` to install protobuf.

Reply:
Brew works for Linux too. This is my attempt to make it easier to install everything. What I'll do is add links for each of the formatters so that the reader can follow the instructions for installing each of them if needed.

Reply:
Moving this to the Protolint PR